templates/blog/index.html

```diff
@@ -1,4 +1,5 @@
 {% extends "_base/base.html" %}
+{% set active_page = "blog" %}
```

Comment:
you need to add this to the article pages too

Reply:
updated